### PR TITLE
feat(card): add subscribe card

### LINKED ISSUE #217: Feature card - 002 [Newsletter Subscription]
### Description
Create the feature card component as shown in the video attached below.

### Animation Preview
- Screenshot/Video: 

https://github.com/user-attachments/assets/4c42c509-43d7-410b-82fd-d9d2475f5b1d

> [!NOTE]
> Please refer to the **Animata contributing guidelines** available [here](https://www.animata.design/docs/contributing) for rules on how to contribute to the project. Let us know in the comments if you’re working on this issue, and feel free to ask any questions!

### Requirements
- [ ] Create a new animated component that matches the provided design.
- [ ] Add example usage of the component in the documentation or storybook.

<details>
  <summary>
    <h3> Guidelines & Best Practices</h3>
  </summary>

- Write reusable and clean code following the project’s style guide.
- The component should be **fully customizable**. The user should be able to modify props such as duration, easing, color, and size where applicable.
- Follow the [Contributing Guidelines](https://www.animata.design/docs/contributing) before submitting your pull request.

</details>

> [!IMPORTANT]  
> To ensure more contributors have the opportunity to participate, we kindly request that:
> - Each contributor submits only **one pull request** related to this issue.
> - If you're interested in working on this issue, first check if it has already been claimed. If not, **assign yourself** to the issue and move it to **In Progress** on the project board.

---
### Checklist
- [ ] I have read and followed the contributing guidelines.
- [ ] I have developed the component according to the requirements.
- [ ] I have tested the component across different browsers and screen sizes.
- [ ] I have documented the component for future reference.
- [ ] I have submitted a pull request with appropriate labels and referenced this issue number.

### PR DESCRIPTION
**Pull request by Prithwi32**
Closes #217

### CODE REVIEW

**Comment by Prithwi32** [comment]
@sudhashrestha could you review PR and let me know any feedback

File: content/docs/card/feature-card.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+<ComponentPreview name="card-feature-card--docs" />
+
+## Installation
+
+<Steps>
+<Step>Install dependencies</Step>
+
+```bash
+npm install shadcn-ui lucide-react
```
Comment (hari) [inline content/docs/card/feature-card.mdx:15]: Remove shadcn-ui as it is not installed like this.

File: content/docs/card/feature-card.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          <p className="pl-4 hover:font-bold">{buttonText}</p>
+        </Button>
+      </div>
+    </div>
+  );
+}
+```
+
+</Steps>
+
```
Comment (hari) [inline content/docs/card/feature-card.mdx:71]: You don't need to add the code here.  Following line will automatically display your code in the docs page.

```
jsx file=<rootDir>/animata/card/feature-card.tsx
```

File: animata/card/feature-card.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  placeholder?: string;
+  buttonText?: string;
+}
+
+export default function FeatureCard({
+  title = "Want to read the rest?",
+  placeholder = "justin@buttondown.email",
+  buttonText = "Subscribe for $5/mo",
+}: FeatureCardProps) {
+  const [isButtonHovered, setIsButtonHovered] = useState(false);
```
Comment (hari) [inline animata/card/feature-card.tsx:18]: You don't need to add the state for `hover`. Simply use `hover:skew-x-0 hover:skew-y-0`

Comment (hari) [reply animata/card/feature-card.tsx:18]: Just to clarify, the skew effect is triggered on hovering the card not just the button.

**Review by hari — CHANGES_REQUESTED** [review]

File: content/docs/card/feature-card.mdx (hunk 2 above)
Comment (Prithwi32) [reply content/docs/card/feature-card.mdx:71]: okay sir

File: animata/card/feature-card.tsx (hunk 3 above)
Comment (Prithwi32) [reply animata/card/feature-card.tsx:18]: > Just to clarify, the skew effect is triggered on hovering the card not just the button.

okay I saw video, there skewing card  when hovering btn, so I done I like sir

Comment (Prithwi32) [reply animata/card/feature-card.tsx:18]: @hari should i change now or okay, skew resetting after button hover looks more good i feel...
If not required i will change sir ... please let me know

Comment (hari) [reply animata/card/feature-card.tsx:18]: @Prithwi32 you can change it.

Comment (Prithwi32) [reply animata/card/feature-card.tsx:18]: Okay

Comment (Prithwi32) [reply animata/card/feature-card.tsx]: Done sir @hari 
check this: 

https://github.com/user-attachments/assets/c60c32f8-be64-4ab8-bcd7-eca2c3b21695

**Review by hari — CHANGES_REQUESTED** [review]
@Prithwi32 Please rename the file to `subscribe-card`. You should update the name and path in the docs as well.

File: content/docs/card/feature-card.mdx [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Feature Card
+description: When the cursor hovers over the button, the card skews and the check mark appears.
+author: PrithwiHegde
+---
+
+<ComponentPreview name="card-feature-card--docs" />
```
Comment (hari) [inline content/docs/card/feature-card.mdx:7]: @Prithwi32 You need to rename this file as well to `subscribe-card.mdx` and update the `feature-card` in the docs to `subscribe-card`.

File: animata/card/subscribe-card.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import FeatureCard from "@/animata/card/feature-card";
```
Comment (hari) [inline animata/card/subscribe-card.stories.tsx:1]: @Prithwi32 The import path is wrong.

File: animata/card/subscribe-card.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import { Meta, StoryObj } from "@storybook/react";
+
+const meta = {
+  title: "Card/Feature Card",
```
Comment (hari) [inline animata/card/subscribe-card.stories.tsx:5]: Change this to `Card/Subscribe Card` as well.

**Review by hari — CHANGES_REQUESTED** [review]

File: animata/card/subscribe-card.stories.tsx (hunk 6 above)
Comment (Prithwi32) [reply animata/card/subscribe-card.stories.tsx:5]: @hari i changed all these now in vscode I will push, but http://[localhost:3000/docs/card/subscribe-card](http://localhost:3000/docs/card/subscribe-card) will work right, its already set when created component, how I should change path ..please help

**Review by hari — APPROVED** [review]

**Comment by hari** [comment]
Looking good @Prithwi32 

Thank you for the patience and contribution!

**Pull request merged**